sympy/solvers/diophantine.py

```diff
@@ -1416,7 +1419,7 @@ def cornacchia(a, b, m):
     >>> cornacchia(2, 3, 35) # equation 2x**2 + 3y**2 = 35
     {(2, 3), (4, 1)}
     >>> cornacchia(1, 1, 25) # equation x**2 + y**2 = 25
-    {(3, 4)}
+    set([(4, 3)])
```

Comment:
This could probably be `{(4, 3)}`, which looks better. And should the other solutions be included?